doc/source/contributor/environments/ci-aio.rst

```diff
@@ -11,8 +11,29 @@ Prerequisites
 * a CentOS Stream 8 or Ubuntu Focal 20.04 host
 * access to the Test Pulp server on SMS lab
 
-Setup
-=====
+Automated Setup
+===============
```

Comment:
Headings don't really make sense here as they're all at the same level.

```
Automated
==

Manual
==

Setup
--

Installation
--
```

Reply:
Done